helm-chart/images/binderhub/requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.8
 # To update, run:
 #
-#    ./dependencies freeze --upgrade
+#    see requirements.in
```

Comment:
Since this is no longer a command the above 4 lines can be changed to
`To update, see requirements.in`

Reply:
This whole file is auto-generated by pip-compile. We only provide the "see requirements.in" bit, in the snipped above

Reply:
The file is autogenerated, but having `CUSTOM_COMPILE_COMMAND="see requirements.in"` when `pip-compile` runs makes this part be autogenerated in the way that makes `./dependencies freeze --upgrade` be replaced by `see requirements.in` I think.

Reply:
I think we can ignore this and make it update to be that automatically in the future as well, just one less point of confusion for the person autogenerating this in the future.

Reply:
It's now `see README.md` where there's a more detailed explanation of pip-compile and friends, including the one-liner to run pip-compile in docker. We can leave it as the default `pip-compile` here if that seems preferable.